fontc/src/lib.rs

```diff
@@ -678,6 +678,47 @@ mod tests {
         assert_eq!(expected, completed);
     }
 
+    /// A convenience type for the common case of "just compile this source with
+    /// default arguments".
+    ///
+    /// When you need more control, just the `compile` fn/`TestCompile` type
+    /// directly
+    struct SimpleCompile {
+        /// we need to hold onto this because when it goes out of scope,
+        /// the directory is deleted.
+        _temp_dir: TempDir,
+        /// the font bytes
+        buf: Vec<u8>,
+        result: TestCompile,
+    }
+
+    impl SimpleCompile {
+        fn new(source: &str) -> std::io::Result<Self> {
+            let temp_dir = tempdir()?;
+            let build_dir = temp_dir.path();
+            let result = compile(Args::for_test(build_dir, source));
```

Comment:
I don't particularly like that the heavy-lifting part of the code where things get compiled is hidden in a default new() constructor.. Why not use a free function that takes a str and returns a struct and is named with something like "compile" to make it clear this is, well, compiling?

Reply:
+1, I like the current `result = compile()` pattern. I also like how this PR removes some of the boilerplate. Seems possible to combine.

Reply:
fair, how about `SimpleCompile::compile`?